style/js/shield_defs.js

```diff
@@ -821,6 +821,8 @@ export function loadShields(shieldImages) {
     null
   );
 
+  shields["US:VI"] = circleShield("white", "black");
```

Comment:
USVI highways are apparently numbered [into the thousands](https://en.wikipedia.org/wiki/List_of_United_States_Virgin_Islands_highways), so we should make sure `circleShield` can accommodate four-digit numbers. If it looks too pointy, `roundedRectShield` would be an alternative, though it would result in a Kentucky-style elongated circle instead of an ellipse for any three- to four-digit shield.

Reply:
I think it looks alright. Why is Kentucky using `roundedRectShield` instead?

![Screenshot from 2022-03-23 22-25-03](https://user-images.githubusercontent.com/1732117/159829714-9e6284d5-2e2d-4f29-b362-f8f2f35b4904.png)

Reply:
We used rounded rects in Kentucky because that's how they're shaped:
![image](https://user-images.githubusercontent.com/3254090/159830523-62fe231d-3be1-4943-abce-0e279c4db9ba.png)

Reply:
Meanwhile in Delaware they use ellipses on their wider shields:
![image](https://user-images.githubusercontent.com/3254090/159830685-63e77bb8-ccc4-4dbb-8a58-c325281cac30.png)

Reply:
Kentucky’s shields are elongated circles (rounded rectangles) in reality, and four-digit route numbers are very common there, so the rounded rectangles would be familiar. That isn’t the case in every state that uses circular shields, so it’s fine to use the ellipses here.

Reply:
Since I'm a super math nerd, I implemented drawing options for both form factors :-D

Reply:
USVI highways apparently use narrower text, keeping a strict circle, as Iowa does. So an ellipse is a reasonable compromise for legibility on this map.

<img src="https://upload.wikimedia.org/wikipedia/commons/0/0d/Circle_sign_104.svg" width="100" alt="USVI 104">